_schedule/talks/2017-08-14-19-00-reception.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: session-details
+category: tutorial
```

Comment:
I'm 90% sure this is ok, but it might need to be `talk`. If it worked for you locally it's fine then.